src/rapidjson/reader.h

```diff
@@ -71,7 +71,7 @@ RAPIDJSON_DIAG_OFF(effc++)
     \ingroup RAPIDJSON_ERRORS
     \brief Macro to indicate a parse error.
     \param parseErrorCode \ref rapidjson::ParseErrorCode of the error
-    \param offset  position of the error in JSON input (\c size_t)
+    \param offset position of the error in JSON input (\c size_t)
```

Comment:
there were a couple of places where things appeared to be intentionally indented. This doesn't appear to be one of those cases...